scripts/preview/preview.js

```diff
@@ -0,0 +1,198 @@
+import path from 'node:path'
+import fs from 'node:fs'
+import yaml from 'yaml'
+import {findUp, pathExists} from 'find-up'
+import { promisify } from 'node:util'
+import child_process from 'node:child_process'
+import deepmerge from '@fastify/deepmerge'
```

Comment:
I think this needs to be listed in the `package.json` file.

Reply:
thanks - I think I added it in the wrong directory, I'll tidy up...